packages/auth/src/core/user/token_manager.ts

```diff
@@ -113,7 +115,7 @@ export class StsTokenManager {
 
   private updateTokensAndExpiration(
     accessToken: string,
-    refreshToken: string,
+    refreshToken: string | null,
```

Comment:
Could you provide some context on this?

Reply:
The user loaded with the `FirebaseServerApp.authIdToken`  invokes `updateFromIdToken` and then `updateTokensAndExpiration` with a `null` valued `refreshToken`. The `refreshToken` is `null` since the user was built from `getAccountInfo`, and therefore doesn't have a `refreshToken`.